packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -80,7 +81,7 @@ const getBaseUrl = (formString: string, chat: boolean = false): URL => {
   )
 }
 
-export const buildIframeSrc = (params: BuildIframeSrcOptions): string => {
+export const buildIframeSrc = async (params: BuildIframeSrcOptions): Promise<string> => {
```

Comment:
I dont really like we are introducing a breaking change to the lib API just because of the hubspot functionality. It could break existing implementations that are using the lib from CDN. Is there another approach?